package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,777 @@
+  for frame in w.timeseries:
+      if frame:
+          water_bridge_presence.append(True)
+      else:
+          water_bridge_presence.append(False)
+  print(sum(water_bridge_presence)/len(water_bridge_presence))
+  # 0.5
+  # Water bridge can be detected in 50% of the time
+
+For further data analysis, it is convenient to process the
+:attr:`~WaterBridgeAnalysis.timeseries` data into a normalized table with the
+:meth:`~WaterBridgeAnalysis.generate_table` method, which creates a new data
+structure :attr:`WaterBridgeAnalysis.table` that contains one row for each
+observation of a hydrogen bond::
+
+  w.generate_table()
+
+This table can then be easily turned into, e.g., a `pandas.DataFrame`_, and
+further analyzed::
+
+  import pandas as pd
+  df = pd.DataFrame.from_records(w.table)
+
+
+.. _pandas.DataFrame: http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.html
+
+Classes
+-------
+
+.. autoclass:: WaterBridgeAnalysis
+   :members:
+
+   .. attribute:: timesteps
+
+      List of the times of each timestep. This can be used together with
+      :attr:`~WaterBridgeAnalysis.timeseries` to find the specific time point
+      of a water bridge existence, or see :attr:`~WaterBridgeAnalysis.table`.
+
+   .. attribute:: table
+
+      A normalised table of the data in
+      :attr:`WaterBridgeAnalysis.timeseries`, generated by
+      :meth:`WaterBridgeAnalysis.generate_table`. It is a
+      :class:`numpy.recarray` with the following columns:
+
+          0. "time"
+          1. "donor_index"
+          2. "acceptor_index"
+          3. "donor_resnm"
+          4. "donor_resid"
+          5. "donor_atom"
+          6. "acceptor_resnm"
+          7. "acceptor_resid"
+          8. "acceptor_atom"
+          9. "distance"
+          10. "angle"
+
+      It takes up more space than :attr:`~WaterBridgeAnalysis.timeseries` but
+      it is easier to analyze and to import into databases or dataframes.
+
+
+      .. rubric:: Example
+
+      For example, to create a `pandas.DataFrame`_ from ``h.table``::
+
+         import pandas as pd
+         df = pd.DataFrame.from_records(w.table)
+"""
+import numpy as np
+import logging
+import warnings
+
+from .hbond_analysis import HydrogenBondAnalysis
+from MDAnalysis.lib.NeighborSearch import AtomNeighborSearch
+from MDAnalysis.lib.log import ProgressMeter, _set_verbose
+from MDAnalysis import SelectionWarning
+
+logger = logging.getLogger('MDAnalysis.analysis.wbridges')
+
+class WaterBridgeAnalysis(HydrogenBondAnalysis):
+    """Perform a water bridge analysis
+
+    The analysis of the trajectory is performed with the
+    :meth:`WaterBridgeAnalysis.run` method. The result is stored in
+    :attr:`WaterBridgeAnalysis.timeseries`. See
+    :meth:`~WaterBridgeAnalysis.run` for the format.
+
+    :class:`WaterBridgeAnalysis` uses the same default atom names as \ :class:`~MDAnalysis.analysis.hbonds.hbond_analysis.HydrogenBondAnalysis`, see :ref:`Default atom names for hydrogen bonding analysis`
```

Comment:
79 characters, PEP 8

Reply:
why a back slash here?

Reply:
@orbeckst Thank you. I find that sometimes :class:`~MDAnalysis.analysis.hbonds.hbond_analysis.HydrogenBondAnalysis` will not generate the correct link and adding a '\ ' before it solves the problem.